### ISSUE TITLE
comoving distance differs from docs

### ISSUE THREAD

**Issue by zingale** [abatten/fruitbat#31]
Working through: 

https://fruitbat.readthedocs.io/en/latest/user_guide/using_fruitbat.html

when I get to "Calculating Distances" and the first subsection, "Comoving Distance", I get 
```
In [22]: frb.calc_comoving_distance()                                                                 
Out[22]: <Quantity 1812.21526231 Mpc>
```
instead of the 2020.297 value reported.  Once I create the next `frb` object in the code block below this, if I go back and look at `calc_comoving_distance()` I get the value in the docs.  So perhaps the docs were executed out of order?

**Comment by abatten** [comment]
Ahh, I think at one point I wrote luminosity distance before the comoving distance and I rearranged then to be in alphabetical order which causes that mistake. This should be a simple fix, I'll add the same code from luminosity distance to the comoving distance section.

**Comment by abatten** [comment]
This has been fixed in #33.
